Lab_2.ipynb

```diff
@@ -0,0 +1,783 @@
uLvpbde3u2XFAw/idTjo9cAfUWazrj4c9a2s/6SArDPiSR9ozKhdY3UVHz7+F2JSUrnk7t8RHvXN6alRCYlc9ps/Eh4dzYKHfk9TTWAFxQNhMinOvHYQrlYPn/9rZ8AjGyabjfRnnsaamkrxzT+mZWfg68YGxQ/itbmvEWuL5aZFN/FJ3icB99lt9MmFHy2DC/4GdYW+wudvXgVlW42OrFsYkBLJ/RcMYc09M3l8/giSo2w8tHAPk/70BTe+vI53N5bQ0OIyOkwhhBAiIJLcdZBOr3GnafDxHYCCuY8EbemDNWVreGrLU8zJnqN7Omb9Bx/Q8PHHxP/4ZkL79tUdy+oFB/C4vEy+rJ/uPgLh9Xj4+MmH8LhcXHDHPYTZjz21NCImlovv/C1tLc18/OTDeD2BlSQIRFyvCKbO70fxzhrWfxx4+QFLXBwZLzyPKTKSwu/fQPPWwBORtMg0Xp3zKkPih3Dn8jt5cM2DuDynyUW5yQyjroWfboTpd0P+ct96vDe+5xvRFydls5q5aGRv3vrRRBbfkcsNU7LZUdrAz9/awpg/fC6JnhBCiG5NkrsO4PJ4Kalt7twad1/9FfYt8u2OGZPeeccJQHFjMXcsvYOsqCzum3CfrlHM1rw8yu7/HWFjRpPwwx/qjqVoRzW7V5czYlYGMcnG1LVb9c4bHNy9k1k/vJW41BPv9BmflsGsG2+hZNd2Vv73310U4bENnpLKwAkprPukgIJtVQH3Z01NJfPVVzBHRlL0/RtwrAl8OmGsLZbnz3meawdfyxu73+D6hddzsOlgwP12G6F2mH4X3LbVl+QVfuUrnfDKRbB3kZRPOEU5iXbumTuIFXeeyTs/nsQ1EzOPmejVykYsQgghugkVDFuxn6oxY8Zo6ztg7U5HK6x2kPvwEh66bBjzxnRC4rVnIbxxBZxxGVzyXFCO2tW21HLt/66lpqWGN899k/Qo//8fPE1NFF75PdxVVWS/twBrcrKuWFqb3bz5+zVYQ83M+/VYLFZ90zoDUbZvD2/c90sGT53B7J/cfsrtPn36SbYv+YzL7vkDmcNGdF6AJ+Fq87DgkY3Ulju48PaRpGQHXkLCVVFB0Q0/oK2wkJR77yV2vv61lEdbVLCI+1beh6Zp3D76duYNmIdJnWZ/t2qph3XPw9rnoLEM4vvC+Jth2DywGVv+o7vxejU2Fdfx8dYy/re9jLL6FpSCYWkx5PZPJLd/IsPTorGYT7NzTAghRNBQSm3QNO2YW+ZLctcBlu49xHUvruWtH01kXHYHr+06tAeemwnxfeCGT8Ea1rH9d4CmtiZuWnQT++r28cysZxidPNrvPjSXi+If3Yxj7Voynn2GiEmTdMWiaRqfPrudvC1VXPKLUaT06foLW1drC6/e+TPcrjaue/jvhIaf+nRdV2sL/77n5zQ3NnDtQ38jIia2EyM9MUd9K+8+vIG2Zg+X/HIUsSmBTzv2NDZy8Od34Fi+nNirriL5rjtR1sA3ISptKuX+lfezqmwVY1PGct+E+8iKzgq4327H44Kd78Oqf0DpRrCEweALYeTVvrp5QfiHoWDm9WpsKalj6d5DLN17iC3FdXg1iLJZmNovkWn9E5jaL5HUmOD7uSyEEKLnkuSuk72yqoD73t/B2ntmkhRl67iOm+vguTOhtQF+uASi9RXx7kz1rfX8+PMfs6t6F3+d/ldmZMzwuw/N46Hsnnuof/8Dej34IDGXXKw7ns2fF7Hi7f1MuqQvI8/O0N1PIL546Rk2LfyQy+99gIyhw/1uX1VUwL/v+Tm9Bw3h0rt/hzIZN0JQV+nk3Yc3AHD+T0eQmKGvJMXRNI+Hykcepeall7ANHUrqX/5MaE5O4P1qGgv2L+DhdQ/T4m5h/sD53DzsZmJsMQH33e1omq8e5qZXYdvbvp8hMZkw9BIYcgmknCGJng51zja+2l/F0j2HWLbvEBUNrQD0jgljbFYso7PiGJsVS/+kSEwm+f8VQgjROSS562S//3Anb6wtYufvz+m43TJbm+DNK6FwFVz3IWRO7Jh+O1C5o5xbFt9CXn0ej+Y+ypkZZ/rdh+ZyUXrX3TR8/DGJt91Gws0/0h3PgU2VLHx2O31GJDL7h0MNKX1QuHUzbz/wG0bNuYAZ1+tfM7h18UI+e/bvTLniWsZf3DHTF/Wqq3Dy/uObaGt2M+fHw0gb0DGjiQ0LP6X8/vvxOp0k3n47cddeo3tn1KNVNVfxz83/5J197xBhjeD7Q77P/IHziQqJ6oCou6E2J+z6ALb+B/KWgubxTdscdAEMmAO9R/s2ahF+0TSNPRWNrNhfzYbCGtYX1FLZ6Ev2Im0WRmfGMiYzlpEZsQxJjSImPMTgiIUQQvQUktx1sh/8ax0H65pZeNu0jumw6RC8frlvi/OLnoLh8zum3w60sWIjv1j6C5rdzTw6/VEmpfo/jdLT2EjpL35J09KlJN7xcxJuukl3PEU7qvnkqW0kZti54LaRWEO6/mK1qaaaV+/6GTZ7JFf/+XGsIaG6+9I0jY+feIi9a1Yw/7d/pvfAwR0Yqf8aa1r48MnN1FU2M/GiHEbMSu+Q5Nl96BBl9/2Wpi+/JLRfP5LuvBP7lMkdEDHsq93H4xsfZ1nJMuxWO/MGzOOqQVeRFJ7UIf13S44q2PUh7HgXClb4Er3weOg7C/qeBdlTITLF6Ci7JU3TKK5pZn1hDesKatlQWMPeiqYjr/eOCWNIahRDe0cfuU2KDDWs/qYQQojuS5K7Tjbz0SX0S4rk6Wv8X2v2HTX58Nol0FAGl/8LBswOvM8O5PK6eHHbizy15SlS7ak8OeNJ+sb6X66g9cABSm79KW1FRaT85tfEXnml7pj2b6jksxd3ENsrgotuG4nN3smF5I/B43bz3z/8mor8/Vz94GPEpwU+JbTV6eTVu/4Pj8vFlX94hKiExA6IVL+2ZjdfvLKLA5sOkTEknulXDSAyLvBpyJqm0fjpIioffRRXcTERU6aQcPOPCBs9ukMufHdV
```

Comment:
You don't seem to quite touch on the "(Clearly state the statistical question, then turn that into math using your background distribution from part 1 of this problem.)" part, unless that was the question you had mentioned above? If so it would probably be best to move these things closer together.

Reply:
you're right, I think I forgot to do it again thanks for catching that